resources/keywords/import_provider_account.resource

```diff
@@ -197,6 +212,8 @@ Provider Account Import Failure
     Provider Account Import Failure Screen
     User Navigates To DBaaS Inventories Screen
     Check Status Of Invalid Provider Account
+    Run Keyword If    "amazon" in """${provaccname}"""
+    ...    Tear Down The Test Suite
```

Comment:
@ikanias @mrrajan 
IMHO, Test Suite tear down should not be invoked within mid-level keywords
This may not be a problem in scenarios where we are expecting a failed import, but if in future this keyword is used at AND level instead of THEN, it may result in ambiguous failures
Moreover once we expand the test Suite to include the scenarios of connecting with Openshift application, we will need to decide where to insert these 2 lines so that each scenario can run independently as well as together.

I propose a slightly different approach -
Include the removal of provider account at Test Case Tear down level only for Amazon RDS and Invalid imported accounts.

```
Tear Down The Test Case
    Log    Test Case Tear Down
    Close Browser
    Run Keyword If    "${instanceName}" == "${EMPTY}"
    ...    Remove Created Namespaces
    Run Keyword If    """inv""" or """amazon""" in """${provaccname}"""
    ...    Remove Imported Provider Accounts
```

With this change, the imported Amazon RDS imported account will get removed during the teardown of the respective rds test case.

Reply:
@harshkumarRH I agree on your suggestion